app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/Database/Workout/WorkoutRepository.java

```diff
@@ -65,10 +109,43 @@ public Workouts call() throws Exception {
         }
 
         @Override
-        protected Void doInBackground(final Workouts... params) {
+        protected Void doInBackground(final Workout... params) {
             mAsyncTaskDao.addWorkout(params[0]);
             return null;
         }
     }
 
+    public void deleteAll()
+    {
+        new DeleteAllAsyncTask(mWorkoutDao).execute();
+    }
+    private static class DeleteAllAsyncTask extends AsyncTask<Void,Void,Void>
```

Comment:
This is not needed for this pull request, but it would be nice if we can find a way to refactor this to avoid somme of this AsyncTask boilerplate. I'm not sure how easy that will be.

Reply:
I agree, when I was writing this I definitely felt the same, but wanted to get this out so Neil would have a basis to go on.  Its definitely something to go back to if we have time.